spec/bugsplat.spec.ts

```diff
@@ -224,7 +224,7 @@ describe('BugSplat', function () {
             'callstack',
             jasmine.stringMatching(
                 new RegExp(
-                    `Error: ${expectedError}.*\n.*at BugSplat\.<anonymous>`
+                    `Error: ${expectedError}.*\n.*at BugSplat.<anonymous>`
```

Comment:
why did this change? i think we want to escape the . here because we're looking for `.` in the call stack specifically whereas in a regex the . means any character.

Reply:
tbh I should have tested on this case further before changing it. I changed this because I was getting an error from it after installing eslint:
![image](https://user-images.githubusercontent.com/26882268/164913380-7b59855b-0e02-42f5-a1c4-ea989ec09a3b.png)

Which is this rule: https://eslint.org/docs/rules/no-useless-escape

I'm not sure my change is correct, but I think the line that it was changed from is not correct either. 

When I run this line:
``new RegExp(`Error: ${expectedError}.*\n.*at BugSplat\.<anonymous>`)``
I get this
``/Error: Error without a stack!.*\n.*at BugSplat.<anonymous>/``

I think what we actually need is:
``new RegExp(`Error: ${expectedError}.*\n.*at BugSplat\\.<anonymous>`)``

As to why, I would have to look more into the compilation of RegExp.

Reply:
I investigated this a little bit and it is happening due to the template string. The template string is compiled before it is passed to the RegExp and all it does is remove the `\` because `\.` isn't a valid escape sequence for template string literals. Then it is passed to the RegExp as ``/Error: Error without a stack!.*\n.*at BugSplat.<anonymous>/``.

So it does look like we have to escape it with `\\` so that the resulting string that is passed to RegExp only has a single `\`.

Reply:
Thanks for looking into this! 😎